### PR TITLE
fix lists docs regression

### PR DESCRIPTION
**Pull request by ringabout**
Before the PR
https://nim-lang.github.io/Nim/lists.html

image

After the PR

image

### CODE REVIEW

File: lib/pure/collections/lists.nim [hunk 1]
```diff
@@ -231,7 +224,7 @@ iterator items*[T](L: SomeLinkedRing[T]): T =
     from std/sugar import collect
     from std/sequtils import toSeq
     let a = collect(initSinglyLinkedRing):
-      for i in 1..3: 10*i
+      for i in 1 .. 3: 10 * i
```
Comment (timotheecour) [inline lib/pure/collections/lists.nim:227]: @xflywind that change is controversial
```nim
1..3
=>
1 .. 3
```

in the past people have done the opposite change; we should maybe not change such instances until there's a guideline for it in `nep1.rst` or `contributing.rst` as a reference guide. My personal preference is `1..3`

Comment (ringabout) [reply lib/pure/collections/lists.nim:227]: I would like
-5 .. -1

Comment (timotheecour) [reply lib/pure/collections/lists.nim:227]: the styleguide rule could be:
use `a..b` except if `b` involves an operator (eg -1)

Comment (timotheecour) [reply lib/pure/collections/lists.nim:227]: => https://github.com/nim-lang/Nim/pull/16992

**Pull request merged**